package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "electron-builder",
   "description": "Complete solution to build ready for distribution and 'auto update' Electron App installers",
+  "version": "0.0.0-semantic-release",
```

Comment:
https://youtrack.jetbrains.com/issue/WEB-20510

Don't want to disturb other contributors.